tmail-backend/jmap/extensions-opensearch/src/main/java/com/linagora/tmail/james/jmap/OpenSearchContactConfiguration.java

```diff
@@ -104,7 +104,7 @@ public static Builder builder() {
     public static final WriteAliasName DEFAULT_ALIAS_WRITE_DOMAIN_CONTACT_NAME = new WriteAliasName("domain_contact_write_alias");
     public static final ReadAliasName DEFAULT_ALIAS_READ_DOMAIN_CONTACT_NAME = new ReadAliasName("domain_contact_read_alias");
     public static final Integer DEFAULT_MAX_NGRAM_DIFF = 27;
-    public static final Integer DEFAULT_MIN_NGRAM = 3;
+    public static final Integer DEFAULT_MIN_NGRAM = 1;
```

Comment:
2 as a minimum please. We are not google.

Reply:
Why 2 is better?

Reply:
> Why 2 is better?

I explained in the commit message: `Reason: to not return too noise suggestions, and save indexing space.`

Reply:
Thanks!